lib/shoot/ngrok.rb

```diff
@@ -0,0 +1,23 @@
+require 'childprocess'
+require 'forwardable'
+require 'securerandom'
+
+module Shoot
+  class Ngrok
+    extend Forwardable
+    def_delegators :@process, :start, :stop, :exited?
+
+    def initialize(port = 3000)
+      @process = ChildProcess.build("ngrok", "-log=stdout", "-subdomain=#{subdomain}", port.to_s)
+      start
```

Comment:
No. The idea of the ChildProcess is exactly that when your process dies, it dies together.